change/@microsoft-fast-foundation-3258c29c-5cb1-4cca-a5c7-fec489832cb1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: AI doesn't update activeid attr is changed",
```

Comment:
Could you update this to match the PR title for the changelog?

Reply:
```suggestion
  "comment": "fix: active indicator does not update when activeid attr is changed",
```